cypress/support/commands.js

```diff
@@ -0,0 +1,111 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+
+Cypress.Commands.add('getNextSlide', { prevSubject: 'optional' }, () => {
+  return cy.get('.swiper-button-next');
+});
+Cypress.Commands.add('getPrevSlide', { prevSubject: 'optional' }, () => {
+  return cy.get('.swiper-button-prev');
+});
+Cypress.Commands.add('nextSlide', { prevSubject: 'optional' }, () => {
+  cy.getNextSlide().click();
+});
+Cypress.Commands.add('prevSlide', { prevSubject: 'optional' }, () => {
+  cy.getPrevSlide().click();
+});
+// subject, options
+Cypress.Commands.add('getActiveSlide', { prevSubject: 'optional' }, () => {
+  return cy.get('.swiper-slide-active');
+});
+
+Cypress.Commands.add('getSliderWrapper', { prevSubject: 'optional' }, () => {
+  return cy.get('.swiper-wrapper');
+});
+
+Cypress.Commands.add('getSliderContainer', { prevSubject: 'optional' }, () => {
+  return cy.get('.swiper-container');
+});
+
+Cypress.Commands.add('getSlide', { prevSubject: 'optional' }, (subject, slideIndex) => {
+  return cy.get(`.swiper-slide:nth-child(${slideIndex})`);
```

Comment:
If we pass slide index which yes, makes sense, then I guess it should be `:nth-child(${slideIndex + 1})` (indexes in CSS starts from `1`) :)

Reply:
Fixed